server/src/sas/CodeZoneManager.ts

```diff
@@ -256,7 +256,7 @@ export class CodeZoneManager {
       type: Token["type"] = "text",
       currLine = line;
     for (; i < len; i++) {
-      if (syntax[i].start >= col) {
+      if (syntax[i].start > col) {
```

Comment:
@jingjiajie, would you please provide a detailed sample about why need to change this?
As it is fundamental logic, I want to verify it carefully. Thank you.